mithril-aggregator/src/services/epoch_service.rs

```diff
@@ -671,82 +722,87 @@ mod tests {
             .collect()
     }
 
-    enum ServiceBuilderParameters {
-        DifferentFixtureForSecondEpoch(MithrilFixture),
-        UpcomingProtocolParameters(ProtocolParameters),
-        WithFutureProtocolParameters(ProtocolParameters),
+    struct EpochServiceBuilder {
+        cardano_transactions_signing_config: CardanoTransactionsSigningConfig,
+        future_protocol_parameters: ProtocolParameters,
+        network: CardanoNetwork,
+        allowed_discriminants: BTreeSet<SignedEntityTypeDiscriminants>,
+        signer_retrieval_epoch: Epoch,
+        next_signer_retrieval_epoch: Epoch,
```

Comment:
Why not store the current epoch and compute these values when building?

Reply:
Done ✅